src/extensions/coblocks-settings/index.js

```diff
@@ -0,0 +1,74 @@
+/*global coblocksSettings*/
+
+/**
+ * Styles
+ */
+import './styles/style.scss';
+
+/**
+ * External dependencies
+ */
+import { PluginMoreMenuItem } from '@wordpress/edit-post';
+import { Fragment, useState } from '@wordpress/element';
+import { registerPlugin, getPlugin, unregisterPlugin } from '@wordpress/plugins';
+import { registerGenericStore } from '@wordpress/data';
+import {
+	__,
+	sprintf,
+} from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import CoBlocksOptionsModal from './coblocks-settings-modal';
+import createCoBlocksStore from './coblocks-settings-store.js';
+
+if ( typeof coblocksSettings === 'undefined' ) {
+	coblocksSettings.coblocksSettings = '0';
+}
+
+if ( coblocksSettings.coblocksSettings === '1' ) {
```

Comment:
Here is the new conditional for the CoBlocks Settings plugin registration. @EvanHerman @jrtashjian Please advise on using a filter to change this setting. A user should be able to add a filter to their theme `functions.php`. Thank you both in advance for your time.